src/routes/conversation/[id]/+server.ts

```diff
@@ -195,6 +195,15 @@ export async function POST({ request, locals, params, getClientAddress }) {
 					updates.push(newUpdate);
 				}
 				controller.enqueue(JSON.stringify(newUpdate) + "\n");
+
+				if (newUpdate.type === "finalAnswer") {
+					// 4096 of spaces to make sure the browser doesn't blocking buffer that holding the response
+					let placeholder = " ".repeat(4096);
+					controller.enqueue(placeholder);
+				}
+				if (newUpdate.token === "") {
```

Comment:
```suggestion
				if (newUpdate.type === "stream" && newUpdate.token === "") {
```

This is needed for ensuring type checking works well, but i'm not sure what this line does anyway since this condition just returns, but we're already at the end of the `update`method ?

Reply:
I found the empty token ("")  will produced when a model targeted on language with using BPE tokenizer. (like Korean)
As the unicode-based language must combine two bytes, byte level bpe can displayed empty token.
(ex. token A + token B = unicode C => {'C', ''})

Reply:
<img width="341" alt="스크린샷 2023-11-20 오후 4 55 24" src="https://github.com/huggingface/chat-ui/assets/11185336/6b8a6c38-7378-4b7d-b821-d5c6016e6b26">

You can see this if you insert 
`console.log("update: " + JSON.stringify(newUpdate));
`
between 202 and 203 line.